index.d.ts

```diff
@@ -309,8 +309,18 @@ declare module '@primer/components' {
   }
 
   export const Dialog: React.FunctionComponent<DialogProps>
+
+  export interface ProgressBarProps
+    extends BaseProps, CommonProps {
```

Comment:
Should this include span attributes as well?
```suggestion
    extends BaseProps, CommonProps, Omit<React.HTMLAttributes<HTMLSpanElement><HTMLAnchorElement>, 'color'> {
```